website/source/docs/auth/gcp.html.md

```diff
@@ -10,106 +10,52 @@ description: |-
 
 # Google Cloud Auth Method
 
-The `gcp` auth method allows Google Cloud Platform (GCP) entities to
-authenticate to Vault. Vault treats GCP as a Trusted Third Party and
-verifies authenticating entities against GCP APIs. This backend allows
-for authentication of:
+The `gcp` auth method allows Google Cloud Platform entities to authenticate to
+Vault. Vault treats Google Cloud as a trusted third party and verifies
+authenticating entities against the Google Cloud APIs. This backend allows for
+authentication of:
 
-* IAM service accounts
-* Google Compute Engine (GCE) instances
+- Google Cloud IAM service accounts
+- Google Compute Engine (GCE) instances
 
-**NOTE**: This backend focuses on identities specific to Google Cloud and
-does not support authenticating Google users (i.e. `type: "authorized_user"`)
-or OAuth against Google
+This backend focuses on identities specific to Google _Cloud_ and does not
+support authenticating arbitrary Google or G Suite users or generic OAuth
+against Google.
 
 This plugin is developed in a separate GitHub repository at
-[`hashicorp/vault-plugin-auth-gcp`](https://github.com/hashicorp/vault-plugin-auth-gcp),
+[hashicorp/vault-plugin-auth-gcp][repo],
 but is automatically bundled in Vault releases. Please file all feature
 requests, bugs, and pull requests specific to the GCP plugin under that
 repository.
 
-## Quick Start
 
-To authenticate, you will need two things:
+## Authenticate
 
-1. Proof-of-identity:
-    * Service Account: A signed [JSON Web Token][jwt] (JWT), or GCP credentials
-    with permissions to sign a JWT (see [`signJwt`][signjwt-method] IAM API
-    method) for this service account
-    * Access to a metadata server for a GCE instance
-1. Name of a `role`, a binding of Vault policies and authZ
-   restrictions preconfigured for the GCP auth backend that
-   an entity authenticates under. Roles are either type `iam`, which
-   supports either type of entity, or `gce`, which support only GCE instances.
+### Via the CLI Helper
 
-### Via CLI
-
-The default path is `/gcp`. If this auth method was enabled at a different
-path, specify `-path=/my-path` in the CLI.
-The backend takes a signed [JSON Web Token][jwt] (JWT) as proof-of-identity.
-
-#### Generate `iam` JWT and Login
-
-Because the process to sign a service account JWT can be tedious, Vault includes
-a CLI helper that handles obtaining a properly-formatted and signed JWT on your
-client before sending an actual request to the Vault server. This process
-**only applies to `iam`-type roles!** For GCE instances, the JWT is
-automatically generated by Google and exposed as metadata.
+Vault includes a CLI helper that obtains a signed JWT locally and sends the
+request to Vault. This helper is only available for IAM-type roles.
 
 ```text
 $ vault login -method=gcp \
     role="my-role" \
     service_account="authenticating-account@my-project.iam.gserviceaccounts.com"
```

Comment:
This isn't changed in this PR, but I do notice it lacks a trailing backslash. Might be nice to add it while we're here.